lib/CreepPlayers/Behaviors/MoveToExitHallwayDumb.js

```diff
@@ -0,0 +1,39 @@
+
+var MoveUtils = require('../../Utils/MoveUtils');
+var Base = require('./Base');
+var Direction = require('../../GameObjects/Direction');
+
+/**
+ *
+ * @param creepController @type CreepController
+ * @constructor
+ */
+function MoveToExitDumb(creepController) {
+    this.cc = creepController;
+}
+
+MoveToExitDumb.prototype = Object.create(Base.prototype);
+
+MoveToExitDumb.prototype.act = function() {
+    var toExit = this.cc.senseDirectionToExit();
+    var toPlayer = this.cc.getDirectionToPlayer();
+
+    var blocking = false;
+    if (!this.cc.canMove(Direction.NORTH) && !this.cc.canMove(Direction.SOUTH) &&
+        this.cc.canMove(Direction.WEST) && this.cc.canMove(Direction.EAST)) {
+        blocking = true;
+    } else if (!this.cc.canMove(Direction.WEST) && !this.cc.canMove(Direction.EAST) &&
+        this.cc.canMove(Direction.NORTH) && this.cc.canMove(Direction.SOUTH)) {
+        blocking = true;
+    }
+
+    //we have a wall on either side, but the direction to the exit is clear.
+    //WE should be blocking the hallway now, waiting for whoever to come through.
+    if (blocking && this.cc.canMove(toExit)) {
```

Comment:
why blocking && this.cc.canMove(toExit)? what if there is a wall in that direction or soemthing?

```
E
WWW
 Y
WWW
```
You are blocking but can't move to exit